source/extensions/tracers/opencensus/opencensus_tracer_impl.cc

```diff
@@ -202,7 +203,7 @@ Driver::Driver(const envoy::config::trace::v2::OpenCensusConfig& oc_config)
   }
   if (oc_config.zipkin_exporter_enabled()) {
     ::opencensus::exporters::trace::ZipkinExporterOptions opts(oc_config.zipkin_url());
-    opts.service_name = oc_config.zipkin_service_name();
+    opts.service_name = local_info_.clusterName();
```

Comment:
Before we merge this: what actually populates `clusterName`?

I have a test config with:
```
static_resources:
  clusters:
  - name: cluster1
```
And I get empty string instead of "cluster1" on L206.

Reply:
It's populated with the local cluster name. This can be set via CLI or in the bootstrap config. https://www.envoyproxy.io/docs/envoy/latest/operations/cli#cmdoption-service-cluster

LMK if this change still looks good to you.

/wait-any

Reply:
Thanks for the tip, setting the name with `--service-cluster` totally works!

I'm happy with this change now.